dynamixel_manipulator:
    # Publish all joint states -----------------------------------
    joints_update:
      type: joint_state_controller/JointStateController
      publish_rate: 50
    
    # Position Controllers ---------------------------------------
    joint_1:
      type: position_controllers/JointPositionController
      joint: joint_1
      pid: {p: 1.0, i: 1.0, d: 0.0}
    joint_2:
      type: position_controllers/JointPositionController
      joint: joint_2
      pid: {p: 1.0, i: 1.0, d: 0.0}
    joint_3:
      type: position_controllers/JointPositionController
      joint: joint_3
      pid: {p: 1.0, i: 1.0, d: 0.0}